tsconfig.esm.json

```diff
@@ -0,0 +1,17 @@
+{
+	"extends": "./node_modules/@dojo/scripts/tsconfig/esm.json",
+	"compilerOptions": {
```

Comment:
seems like the only options that would need adding here are the jsx ones and the types?

Reply:
I added `noUnusedLocals` to the base config and removed the stuff not specific to widget-core